app/assets/stylesheets/application.css

```diff
@@ -1500,6 +1500,15 @@ input[type="submit"].link_post.pushover_button {
 }
 
 /* search */
+@media only screen and (max-width: 450px) {
```

Comment:
Please match the tab indentation used in the rest of the file.

I'm still curious about why this needs a new breakpoint that's so close to the existing one.